app/viewmodels/ErrorMessageAwareness.scala

```diff
@@ -25,5 +25,10 @@ trait ErrorMessageAwareness {
 
   def errorMessage(field: Field)(implicit messages: Messages): Option[ErrorMessage] =
     field.error
-      .map(err => ErrorMessage(content = Text(messages(err.message, err.args: _*))))
+      .map(
```

Comment:
Fix working for most places.
Pages using component `YesNoRadios` still has a custom implementation. This prints Gwall/Error without the `:` like other pages.
https://github.com/hmrc/advance-valuation-rulings-frontend/blob/e6d1609fb03bfb921ae420398c5517e916c39f77/app/views/components/YesNoRadios.scala.html#L48-L60